an-der-uni/weitere.tex

```diff
@@ -8,3 +8,38 @@ \subsection*{Upstream Mathematik-Mentorinnenprogramm}
 
 \subsection*{Beratungsangebote für behinderte Studierende}
 Ganz viele Informationen zum Studium mit Behinderung findet ihr zum einen auf den Seiten der Uni\footnote{\url{http://www.uni-heidelberg.de/studium/kontakt/handicap/}}, zum anderen im Dschungelbuch des StuRa\footnote{\url{https://dschungelbuch.stura.uni-heidelberg.de/index.php/Behinderte_studieren}}.
+
+\subsection*{Kooperationsabkommen mit der Universität Mannheim}
+Zwischen der Universität Heidelberg und der Universität Mannheim 
+gibt es ein Kooperationsabkommen, was Studierenden in Mathematik, Informatik
+und Physik ermöglicht, kostenlos als Gaststudent Vorlesungen an Fakultät für Wirtschaftsmathematik
+und Wirtschaftsinformatik zu hören und sich hier in Heidelberg anrechnen lassen.
+Unter anderem werden in Mannheim Mathematik-Vorlesungen in Kryptographie, 
+angewandte Algebra, Spieltheorie usw. angeboten.
+
+Dieses Kooperationsabkommen bietet Vorteile für die Studierenden, die in der Nähe von Mannheim wohnen,
+und/oder um Vorlesungen zu hören, die es in Heidelberg nicht (mehr) gibt oder unregelmäßig angeboten werden.
+Informatiker können auch die dortigen Grundlagenvorlesungen in Mathematik hören und sich für das Grundstudium
+anrechnen lassen.
+
+Im Gegensatz zu Heidelberg fängt aufgrund der internationalen Ausrichtung der Uni Mannheim das Sommersemester als Frühling-Sommer-Semester
+Mitte Februar und das Wintersemester als Herbst-Winter-Semester Mitte September statt.
```

Comment:
Außerdem sollte sich an die Konvention gehalten werden, dass alle Absätze in einer Zeile stehen.